distributed/cli/dask_worker.py

```diff
@@ -60,15 +60,15 @@
 @click.option('--interface', type=str, default=None,
               help="Network interface like 'eth0' or 'ib0'")
 @click.option('--nthreads', type=int, default=0,
-              help="Number of threads per process.")
+              help="Number of threads in the pool per process.")
 @click.option('--nprocs', type=int, default=1,
-              help="Number of worker processes.  Defaults to one.")
+              help="Number of worker processes to spawn.  Defaults to one.")
```

Comment:
Sometimes we spawn, sometimes we fork, it might be better to avoid using this term explicitly.

Reply:
I thought fork+exec is one way of spawning. `to launch`?